modules/core/server/views/partials/footer.server.view.html

```diff
@@ -7,28 +7,28 @@
   <div class="row">
     <div class="col-xs-3">
       <ul class="list-unstyled">
-        <li><a ui-sref="volunteering">Volunteering</a></li>
+        <!--         <li><a ui-sref="volunteering">Volunteering</a></li>
```

Comment:
I would just delete rather than comment.

Reply:
We should probably move this to a discussion? 
https://github.com/OpenHospitalityNetwork/fedi-trustroots/pull/44#discussion_r720706320